frontend/www/js/omegaup/components/problem/FilterTags.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="card">
+    <div class="card-body">
+      <h3>{{ T.problemEditAddTags }}</h3>
+      <div v-for="(tag, index) in tags" :key="index" class="form-check">
+        <label class="form-check-label">
+          <input
+            v-model="localSelectedTags"
+            :value="tag"
+            class="form-check-input"
+            type="checkbox"
+          />{{ T[tag] }}
+        </label>
+      </div>
+      <div class="form-group">
+        <vue-typeahead-bootstrap
+          :data="publicTags"
+          :serializer="publicTagsSerializer"
+          :placeholder="T.collecionOtherTags"
+          @hit="addOtherTag"
+        >
+        </vue-typeahead-bootstrap>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
```

Comment:
Ya no se utilizarán watchers aquí:
```suggestion
import { Vue, Component, Prop } from 'vue-property-decorator';
```